### ISSUE TITLE
Keep getting "This account may not be used to pay transaction fees"

### ISSUE THREAD

**Issue by syrafs** [solana-playground/solana-playground#80]
Hi, thanks for the awesome tool! 

I may be wrong in the way that I use the playground but after one or two deployments/upgrades, I started to get this error. 

`Create buffer error:  failed to send transaction: Transaction simulation failed: This account may not be used to pay transaction fees`

My workaround is to create a new wallet every time I hit this error and re-deploy the program again or to create a new program credentials before deploying again.

Thanks!

**Comment by acheroncrypto** [comment]
Hey thank you! I've answered why this is happening in https://github.com/solana-playground/solana-playground/issues/74#issuecomment-1303455173.

TL;DR is Anchor doesn't block initializing your own wallet as a program account but it should be fixed in the upcoming Anchor version.

**Comment by syrafs** [comment]
Thanks for the reply @acheroncrypto, appreciate it.

So sorry for not checking the closed issue tab before asking this. Keep up the good work!
